lsp-ui-doc.el

```diff
@@ -86,11 +87,6 @@ This only takes effect when `lsp-ui-doc-position' is 'top or 'bottom."
                  (const :tag "Window" window))
   :group 'lsp-ui-doc)
 
-(defcustom lsp-ui-doc-border "white"
-  "Border color of the frame."
-  :type 'color
-  :group 'lsp-ui-doc)
```

Comment:
Can we keep this option?
Sometime the doc frame is not visible enough, especially when display near caret, so having doc border is helping much more for visibility.
You can set it default to nil though.

Reply:
I had to use the border to pad out the frame, since there were issues with the bottom of the frame having 0 padding. If you have any suggestions to pad out the bottom without the border, I'm all ears.

Reply:
Is the bottom without padding a problem?

Here is what it looks like without border
![image](https://user-images.githubusercontent.com/2631472/85086618-12a43f00-b216-11ea-864f-e31335f15857.png)
and with border.
![image](https://user-images.githubusercontent.com/2631472/85086601-07e9aa00-b216-11ea-92d5-d738cc74a1b2.png)

I would argue that the second one is nicer and more distinguish between popup content and buffer content.

I agree without left/right padding, the popup will look ugly. But that can be solved with `left-fringe`, `right-fringe`. For the top padding, it should be character height already. The bottom padding, is that really necessary?

Reply:
Unfortunately, that's not the result - it ends up looking like this, where the bottom border is directly under the text with no padding at all.
![lsp-ui-doc popup with no bottom padding](https://i.imgur.com/TtOZ7bA.png)

Reply:
I mean, choose between having bottom padding but no border (and hard to distinguish between popup content and buffer content), and no bottom padding but there can be border, the later seems better?

You can take a look again at my screenshot, the one doesn't have border is way to hard to see the content.
Even with the bottom padding, the bottom text line can be displayed with just top-half of the text, and without any border, which makes it looks even weirder.

My point is we should not taken away customization, it will break user configurations. If you still want to have padding for all edges, then probably only do it when `lsp-ui-doc-border` is set to `nil`?
It will archive the same look with when there's padding (and no border), but can still retain the old look of having border, but without bottom padding

Reply:
Yeah, I'd agree with that. I'll push a change to bring the border colour options back. It's probably worth going over to posframe and seeing if they can add some way to specify top and bottom padding, since that'd probably be a useful feature to have there anyway.